staging/src/k8s.io/client-go/testing/fake.go

```diff
@@ -131,13 +131,15 @@ func (c *Fake) Invokes(action Action, defaultReturnObj runtime.Object) (runtime.
 	c.Lock()
 	defer c.Unlock()
 
-	c.actions = append(c.actions, action.DeepCopy())
+	actionCopy := action.DeepCopy()
+	c.actions = append(c.actions, actionCopy)
 	for _, reactor := range c.ReactionChain {
+		// test 'Handles' against the original, unmodified action.
```

Comment:
I decided to keep this as-is to avoid difficult-to-debug errors, and to preserve the current behaviour.

Reply:
yeah, let's consistently pass the same object to Handles and React

Reply:
done 😄